source/fundamentals/aggregation/aggregation-expression-operators.txt

```diff
@@ -0,0 +1,32 @@
+=================================
```

Comment:
Question: 

Did you mean to commit this file? It looks like mostly a duplicate of [aggregation-expression-operations.txt](https://github.com/mongodb/docs-java/pull/330/files#diff-3efcbcb27373cfe178572891a6efd03125e78706f1a6232fa904ec73df71ff5f)

Currently, it's generating a warning because it uses the driver-short constant.

Reply:
deleted this file. I think this issue emerged when i merged master into this branch